code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -309,41 +309,6 @@
 	desc = "A small bottle containing Bio Virus Antidote Kit."
 	list_reagents = list(/datum/reagent/vaccine/fungal_tb = 30)
 
-/obj/item/reagent_containers/glass/bottle/flymutation
-	name = "ancient bottle"
-	desc = "An old bottle with a faded label from eras gone-by. You can't tell if the label has worn away natrually or if it has been hastily scratched off."
-	list_reagents = list(/datum/reagent/mutationtoxin/fly = 5)
-
-/obj/item/reagent_containers/glass/bottle/catmutation
-	name = "ancient bottle"
-	desc = "An old bottle with a faded label from eras gone-by. You can't tell if the label has worn away natrually or if it has been hastily scratched off."
-	list_reagents = list(/datum/reagent/mutationtoxin/felinid = 5)
-
-/obj/item/reagent_containers/glass/bottle/skeletonmutation
-	name = "ancient bottle"
-	desc = "An old bottle with a faded label from eras gone-by. You can't tell if the label has worn away natrually or if it has been hastily scratched off."
-	list_reagents = list(/datum/reagent/mutationtoxin/skeleton = 5)
-
-/obj/item/reagent_containers/glass/bottle/lizardmutation
```

Comment:
This is already available via xenobio, skeleton mutation toxin was probably fine as well.